.github/CODEOWNERS

```diff
@@ -9,3 +9,19 @@
 /master   @determined-ai/backend
 /agent    @determined-ai/backend
 /proto    @determined-ai/backend
+
+# Harness is mapped to ml-sys by default, except for `cli`, `deploy`,
+# and auto-generated bindings and version files.
+/harness  @determined-ai/ml-sys
+/harness/determined/deploy
+/harness/determined/cli
```

Comment:
Echoing Slack, I think ml-sys is also a good owner for CLI (code and tests) and det deploy.

Reply:
already done